packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -200,8 +232,19 @@ export default {
         };
     },
 
+    watch: {
+        immediate: true,
+        isAuthFinished () {
+            if (this.isAuthFinished) {
+                this.initialise();
+            }
+        }
+    },
+
     mounted () {
-        this.initialise();
+        if (this.isAuthFinished) {
```

Comment:
Again, unsure why this is showing up as a change because it is the same as upstream.